configure.ac

```diff
@@ -2,20 +2,26 @@ AC_PREREQ([2.5])
 ##########################################################################
 # ODP API version
 ##########################################################################
-m4_define([odpapi_generation_version], [1])
-m4_define([odpapi_major_version], [37])
-m4_define([odpapi_minor_version], [1])
-m4_define([odpapi_point_version], [0])
-m4_define([odpapi_version],
-    [odpapi_generation_version.odpapi_major_version.odpapi_minor_version.odpapi_point_version])
-AC_INIT([OpenDataPlane],[odpapi_version],[odp@lists.opendataplane.org])
-
-ODP_VERSION_API_GENERATION=odpapi_generation_version
+m4_define([odp_version_generation], [1])
+m4_define([odp_version_major],     [37])
+m4_define([odp_version_minor],      [1])
+m4_define([odp_version_point],      [0])
+
+m4_define([odp_version_api],
+    [odp_version_generation.odp_version_major.odp_version_minor])
+m4_define([odp_version_library],
+    [odp_version_generation.odp_version_major.odp_version_minor.odp_version_point])
```

Comment:
It is a bit dangerous to use the API versioning for libtool versioning (ABI combatibility might break without API combatibility breaking), would be safer to use the separate current:revision:age versioning.